lib/razorpay/item.rb

```diff
@@ -0,0 +1,34 @@
+require 'razorpay/request'
+require 'razorpay/entity'
+
+module Razorpay
+  # Item API allows you to create and fetch customers on Razorpay
+  class Item < Entity
+    def self.request
+      Razorpay::Request.new('items')
+    end
+
+    def self.create(options)
+      request.create options
+    end
+
+    def self.fetch(id)
+      request.fetch id
+    end
+
+    def self.edit(id, options = {})
+      if(!options.is_a?(String) && options.key?(:active))
+        options[:active] = (options[:active] ? 1 : 0)
+      end 
```

Comment:
Why needed? based on docs, they are expected to send only true or false. Dont think we should transpose

Reply:
in some sdk like php and python its working fine but in node and ruby when user put `true/false` it shows message `he active field must be true or false.` .  i used same method for [node](https://github.com/razorpay/razorpay-node/blob/master/lib/resources/items.js#L80)